app/models/service.rb

```diff
@@ -87,7 +87,8 @@ class Service < ApplicationRecord
   virtual_column :power_state,  :type => :string
   virtual_column :power_status, :type => :string
 
-  validates :name, :presence => true
+  validates :name, :presence => true, :length => {:maximum => 256}
+  validates :description, :length => {:maximum => 1024}
```

Comment:
1kb for a service description feels really small.  I'm surprised the UI only allows that much.